src/interfaces/Grant.ts

```diff
@@ -1,21 +1,75 @@
+import { GovernanceApiResponse } from './Api'
 import { TokenSymbols } from '../entities/Tokens'
 import { Category } from './GovernanceProposal'
 import { ProposalParsed } from './Proposal'
 
-interface Grant {
+export interface Update {
+  done_updates: number
+  late_updates: number
+  missed_updates: number
+  update_status?: UpdateStatus
+  health?: ProjectHealth
+  last_update?: string
+  next_update?: string
+  pending_updates: number
+}
+
+export interface VestingInfo {
+  token: TokenSymbols
+  vesting_released: number
+  vesting_releasable: number
+  vesting_start_at: string
+  vesting_finish_at: string
+  vesting_token_contract_balance: number
+  vesting_total_amount: number
+}
+
+export interface OneTimePaymentInfo {
+  token: TokenSymbols
+  tx_date: string
+  tx_amount: number
+}
+
+type Grant = Partial<Update> & Partial<VestingInfo> & Partial<OneTimePaymentInfo> & {
   category?: Category
   tier?: string
   size?: number
   beneficiary?: string
-  token?: TokenSymbols
-  vesting_released?: number
-  vesting_releasable?: number
-  vesting_start_at?: Date
-  vesting_finish_at?: Date
-  vesting_token_contract_balance?: number
-  vesting_total_amount?: number
-  tx_date?: Date
-  tx_amount?: number
 }
 
-export type GrantProposal = Grant & ProposalParsed
\ No newline at end of file
+export type GrantProposal = Grant & ProposalParsed
+
+export type GrantUpdate = {
```

Comment:
It'd be great to import these types from a governance package somehow 🤔

Reply:
That would be great, but I don't know how 🤔